README.md

```diff
@@ -0,0 +1,47 @@
+o-normalise [![Circle CI](https://circleci.com/gh/Financial-Times/o-normalise/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-normalise/tree/master)
+=================
+
+Foundation styles and standardised utilities
+
+- [Usage](#usage)
+	- [Sass](#sass)
+- [Contributing](#contributing)
+- [Contact](#contact)
+- [Licence](#licence)
+
+
+## Usage
+
+### Sass
+
+As with all Origami components, o-normalise has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-normalise-is-silent : false;` in your Sass before you import the o-normalise Sass.
+
+#### Available mixins
+
+- `oNormaliseVisuallyHidden` - provides styles to visually hide an element while remaining accessible to screen readers
+- `oNormaliseClearfix` - adds clearfix styles to an element
+- `oNormaliseBoxSizing` - adds `box-sizing: border-box` to the current and all descending elements
+- `oNormaliseHTML` - adds general normalising styles to `html` and `body` elements
+
```

Comment:
looks like this list is missing some mixins too, like oNormaliseText, oNormaliseLinks, oNormaliseForms and oNormaliseImages